examples/arco-pro/rspack.config.js

```diff
@@ -34,6 +34,55 @@ const config = {
         test: /\.png$/,
         type: 'asset',
       },
+      {
+        test: /\.(j|t)s$/,
+        exclude: [/[\\/]node_modules[\\/]/],
+        loader: 'builtin:swc-loader',
```

Comment:
experiments.rspackFuture.DisableTransformByDefault is set to true since 0.4.0, so you need to transform all module by builtin:swc-loader your self now.